docs/content/recipes/common/replace-crio-runtime.md

```diff
@@ -0,0 +1,163 @@
+# Replacing the Worker Node's CRI-O Runtime
+
+Starting with OpenShift version 4.18, the Machine Config Operator (MCO) will switch the default runtime from `runc` to `crun`, simplying so much the internals for the simplicity, is a C implementation of runc. You can find more information about this change in another section [ADD SECTION HERE!!!!]. In the meantime, this guide will show you how to change the default runtime in Hosted Control Planes (HCP).
+
+## Switching from `runc` to `crun`
+
+The most common use case is when someone using OpenShift 4.16 or 4.17 wants to test their workload performance with `crun` before upgrading to 4.18, where `crun` will become the default runtime.
+
+Ensure you follow these steps to make the change.
+
+### Creating the MachineConfig patch for the NodePool
+
+The following manifest sets `crun` as the default runtime.
+
+=== "**HCP Regular Workloads (CRUN)**"
+
+    ```bash
+    cat <<EOF > mc-crun-default-runtime.yaml
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: master
+      name: 60-crun-default-runtime
+    spec:
+      config:
+        ignition:
+          version: 3.2.0
+        storage:
+          files:
+          - contents:
+              source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lPWNydW4KCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0KcnVudGltZV9wYXRoID0gL2Jpbi9jcnVuCnJ1bnRpbWVfdHlwZSA9IG9jaQpydW50aW1lX3Jvb3QgPSAvcnVuL2NydW4K
```

Comment:
we should first write the actual content of this file to a temp file and then include its base64 encoded version here (maybe as a substitution env var), otherwise it's like a magic incantation.